receiver/splunkenterprisereceiver/scraper.go

```diff
@@ -1560,3 +1561,56 @@ func (s *splunkScraper) scrapeIntrospectionQueuesBytes(ctx context.Context, now
 		s.mb.RecordSplunkServerIntrospectionQueuesCurrentBytesDataPoint(now, currentQueueSizeBytes, name)
 	}
 }
+
+// Scrape introspection kv store status
+func (s *splunkScraper) scrapeKVStoreStatus(ctx context.Context, now pcommon.Timestamp, errs chan error) {
+	if !s.conf.MetricsBuilderConfig.Metrics.SplunkKvstoreStatus.Enabled ||
+		!s.conf.MetricsBuilderConfig.Metrics.SplunkKvstoreReplicationStatus.Enabled ||
+		!s.conf.MetricsBuilderConfig.Metrics.SplunkKvstoreBackupStatus.Enabled ||
+		!s.splunkClient.isConfigured(typeCm) {
+		return
+	}
+
+	ctx = context.WithValue(ctx, endpointType("type"), typeCm)
+	var kvs KVStoreStatus
+
+	ept := apiDict[`SplunkKVStoreStatus`]
+
+	req, err := s.splunkClient.createAPIRequest(ctx, ept)
+	if err != nil {
+		errs <- err
+		return
+	}
+
+	res, err := s.splunkClient.makeRequest(req)
+	if err != nil {
+		errs <- err
+		return
+	}
+	defer res.Body.Close()
+
+	body, err := io.ReadAll(res.Body)
+	if err != nil {
+		errs <- err
+		return
+	}
+
+	err = json.Unmarshal(body, &kvs)
```

Comment:
If you wanted, you can simplify this into:

```golang 
if err := json.NewDecoder(resp.Body).Decode(&result); err != nil {
   errs <- err
   return
}
```

Reply:
I made this change and will probably repeat the pattern for the rest of the scrapers in a separate PR. Thanks!